package.json

```diff
@@ -19,12 +19,13 @@
     "lint": "npm-run-all --parallel lint:*",
     "postinstall": "lerna bootstrap",
     "pretest": "npm run lint && node scripts/check-imports.js",
-    "test": "npm run test:unit && npm run test:closure && npm run test:dependency && npm run build; npm run clean",
+    "test": "npm run test:buildconfig && npm run test:unit && npm run test:closure && npm run test:dependency && npm run build && npm run clean",
```

Comment:
Move this to the end of the tests. Most people just run Unit tests and then force quit because they don't want to wait (at least thats what happens to me)

Reply:
`test:buildconfig` takes less than 1 second to run.

I originally had it at the end, but I got tired of waiting for `test:unit` to finish :grinning:

Reply:
People shouldn't be force-quitting after running unit tests though? At least, not if they want to make sure that everything actually works... (And if they really only want to run unit tests... `npm run test:unit`)

There might actually be solid argument for putting the fast-running things (e.g. `test:buildconfig` and `build`) first since if we manage to do something silly that fails fast, it'd be better for Travis CI to get it over with quick and move on to other jobs. We can do that separately from this PR though.